.travis.yml

```diff
@@ -13,19 +13,7 @@ addons:
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
-  - "7"
-  - "8"
-
-install:
-  - PATH="`npm bin`:`npm bin -g`:$PATH"
-  # Install dependencies and build
-  - npm install
-
-script:
-  # Output useful info for debugging
-  - node --version
-  - npm --version
-  # Run tests
-  - npm test
+  - '12'
+  - '10'
+  - '8'
+  - '8.6.0'
```

Comment:
Doesn't need to be in this PR, but I would love to switch over to GH Actions instead of AppVeyor/Travis.

Reply:
I added GH Actions on the master branch. Will need some tweaking for more Node versions once this is merged (for i.e. Node 12).

Reply:
Neat, will complete in a follow up PR 👍